srcpkgs/deheader/patches/0003-Typo-fix.patch

```diff
@@ -0,0 +1,25 @@
+From b37e3523fe7d70edf418aad7c70c4a3433562676 Mon Sep 17 00:00:00 2001
+From: "Eric S. Raymond" <esr@thyrsus.com>
+Date: Tue, 17 Apr 2018 19:34:55 -0400
+Subject: [PATCH 3/3] Typo fix.
+
+---
+ deheader.xml | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git deheader.xml deheader.xml
+index 1db58a9..549bf1f 100644
+--- deheader.xml
++++ deheader.xml
+@@ -192,7 +192,7 @@ spot. <command>deheader</command> has an internal table of rules that
+ heads off the most common problems by suppressing deletion of headers
+ that are required for portability, but your mileage may vary.</para>
+ 
+-<para>The depenedency scanner does not ignore the text of comments.
++<para>The dependency scanner does not ignore the text of comments.
+ This, e.g, a reference to "log10" in a comment will produce a spurious
+ warning that &lt;math.h&gt;  is required for portability.</para>
+ 
+-- 
+2.20.1
```

Comment:
Can't this patch come from upstream at the next release? Not exactly critical bugfix for functionality.